mmv1/third_party/terraform/resources/resource_composer_environment.go.erb

```diff
@@ -461,6 +460,36 @@ func resourceComposerEnvironment() *schema.Resource {
 								},
 							},
 						},
+						"maintenance_window": {
+							Type:         schema.TypeList,
+							Optional:     true,
+							Computed:     false,
```

Comment:
The failing tests indicate that this field is set by the API even when not set by the user. I think we want this to be `Computed: true`
```suggestion
							Computed:      true,
```

Reply:
done.